src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -172,9 +181,37 @@
                 this.selectFirstItem();
             }
 
+            this.fitItems();
             this.fireValueChangedEvent();
         }
 
+        fitItems() {
+            let itemsWidth = 0;
+            let numberOfOverflowItems = 0;
+            const selectedItemsContainer = this.container.querySelector(SELECTOR_SELECTION_INFO);
+            const selectedItems = this.container.querySelectorAll(SELECTOR_SELECTED_ITEM_IN_LABEL);
+            const selectedItemsOverflow = this.container.querySelector(SELECTOR_OVERFLOW_ITEM_INFO);
+
+            if (selectedItemsOverflow) {
+                selectedItems.forEach(item => { item.hidden = false; })
+                selectedItems.forEach(item => {
+                    itemsWidth += item.offsetWidth;
+
+                    if (itemsWidth > (selectedItemsContainer.offsetWidth - 120)) {
```

Comment:
Why 120? Some comment, it's magic number at the moment

Reply:
Comments in code are bad, assign this to meaningful variable :)